src/mongo/jobs/ApplyOperation.class.php

```diff
@@ -43,6 +43,24 @@ public function perform()
     }
 
     /**
+     * @param \Tripod\Mongo\ImpactedSubject $subject
+     * @param string|null $queueName
+     */
+    public function createJob(\Tripod\Mongo\ImpactedSubject $subject, $queueName=null)
+    {
+        if(!$queueName)
+        {
+            $queueName = \Tripod\Mongo\Config::getApplyQueueName();
+        }
+        $data = array(
+            "subject"=>$subject->toArray(),
+            "tripodConfig"=>\Tripod\Mongo\Config::getConfig()
+        );
+
+        $this->submitJob($queueName,get_class($this),$data);
```

Comment:
I've noticed it in the other branch - so maybe is should go there - but tracing it back up the path there are x3 submitJob functions that look identical. one to refactor at some stage?